CHANGELOG.md

```diff
@@ -13,6 +13,7 @@
 - [Backup NG Overview] List the Backup NG job's modes [#3169](https://github.com/vatesfr/xen-orchestra/issues/3169) (PR [#3277](https://github.com/vatesfr/xen-orchestra/pull/3277))
 - [Backup NG form] Move "Use compression" checkbox in the advanced settings [#2711](https://github.com/vatesfr/xen-orchestra/issues/2711) (PR [#3281](https://github.com/vatesfr/xen-orchestra/pull/3281))
 - [Backup NG form] Ability to remove previous backups first before backup the VMs [#3212](https://github.com/vatesfr/xen-orchestra/issues/3212) (PR [#3260](https://github.com/vatesfr/xen-orchestra/pull/3260))
+- [VM / disks / Attach disk] Display vdi's sr and container in VDI selector  [3021](https://github.com/vatesfr/xen-orchestra/issues/3021) (PR [#3281](https://github.com/vatesfr/xen-orchestra/pull/3285))
```

Comment:
If you want to add a context prefix, it should be more like `[VDI]` as it affects all the VDIs rendered with `renderXoItem`.